server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -361,6 +365,7 @@ public void getSwaggerShouldReturnJsonDataForSwagger() {
             .body(containsString("\"tags\":[\"Address Forwards\"]"))
             .body(containsString("\"tags\":[\"Address Aliases\"]"))
             .body(containsString("\"tags\":[\"Address Groups\"]"))
+            .body(containsString("\"tags\":[\"Cassandra Data\"]"))
```

Comment:
Projects are badly named (`data` is really meaningless) but they are only exposed to developers. Could we avoid leaking these bad names to users ?